pkg/network/ebpf/c/prebuilt/usm.c

```diff
@@ -448,7 +462,11 @@ int uprobe__gnutls_record_send(struct pt_regs *ctx) {
     args.buf = (void *)PT_REGS_PARM2(ctx);
     u64 pid_tgid = bpf_get_current_pid_tgid();
     log_debug("uprobe/gnutls_record_send: pid=%llu ctx=%llx\n", pid_tgid, args.ctx);
-    bpf_map_update_with_telemetry(ssl_write_args, &pid_tgid, &args, BPF_ANY);
+    // buffer could be NULL during non-blocking socket mode, asynchronous gnutls mode
```

Comment:
// buffer could be NULL for the non-blocking asynchronous socket mode used by GnuTLS

Reply:
done